apps/01PigLatin.js

```diff
@@ -3,49 +3,83 @@
 var assert = require('assert');
 var readline = require('readline');
 var rl = readline.createInterface({
-  input: process.stdin,
-  output: process.stdout
+    input: process.stdin,
+    output: process.stdout
 });
 
 
 function pigLatin(word) {
 
-  // Your code here
+    var firstLetter = word[0];
```

Comment:
Even though you know what you are doing now, it will be helpful for you to add comments so that if you look at this code in the future, you will be able to understand what you were coding.